src/js/components/Diagram/doc.js

```diff
@@ -64,6 +74,11 @@ export const themeDoc = {
     type: 'string | {dark: string, light: string}',
     defaultValue: 'accent-1',
   },
+  'global.animation': {
+    description: 'The animation configuration for Diagram.',
```

Comment:
Does this global animation configuration apply to component in addition to Diagram? If so, should this description be modified?

Reply:
Updated this into `diagram.animation` now for more accuracy so it only relates to items related to Diagram's animations.